homeassistant/components/camera/__init__.py

```diff
@@ -102,6 +102,7 @@ class CameraEntityFeature(IntFlag):
 
     ON_OFF = 1
     STREAM = 2
+    STREAM_SNAPSHOT = 4
```

Comment:
The `STREAM_SNAPSHOT `feature is just used to signal that the Camera subclass has an `async_camera_image` that takes a `wait_for_next_keyframe` argument